sdk/python/foundation-models/system/inference/fill-mask/fill-mask-batch-endpoint.ipynb

```diff
@@ -0,0 +1,470 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fill Mask Inference using Batch Endpoints\n",
+    "\n",
+    "This sample shows how deploy `fill-mask` type models to a batch endpoint for inference.\n",
+    "\n",
+    "### Task\n",
+    "`fill-mask` task is about predicting masked words in a sentence. Models that perform this have a good understanding of the language structure and domain of the dataset of they are trained on. `fill-mask` models are typically used as foundation models for more scenario oriented tasks such as `text-classification` or `token-classification`.\n",
+    "\n",
+    "### Model\n",
+    "Models that can perform the `fill-mask` task are tagged with `task: fill-mask`. We will use the `bert-base-uncased` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name. If you don't find a model that suits your scenario or domain, you can discover and [import models from HuggingFace hub](../../import/import-model-from-huggingface.ipynb) and then use them for inference. \n",
+    "\n",
+    "### Inference data\n",
+    "We will use the [book corpus](https://huggingface.co/datasets/bookcorpus) dataset. A copy of this dataset is available in the [book-corpus-dataset](./book-corpus-dataset/) folder. \n",
+    "\n",
+    "### Outline\n",
+    "* Setup pre-requisites.\n",
+    "* Pick a model to deploy.\n",
+    "* Prepare data for inference. \n",
+    "* Deploy the model for batch inference.\n",
+    "* Run a batch inference job.\n",
+    "* Review inference predictions.\n",
+    "* Clean up resources."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Setup pre-requisites\n",
+    "* Install dependencies\n",
+    "* Connect to AzureML Workspace. Learn more at [set up SDK authentication](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-setup-authentication?tabs=sdk). Replace  `<WORKSPACE_NAME>`, `<RESOURCE_GROUP>` and `<SUBSCRIPTION_ID>` below.\n",
+    "* Check or create compute.\n",
```

Comment:
The table of contents here says 'Check or create compute', but the actual instructions are only for creating compute. Do you want to add a block to GET whatever pre-created compute cluster the customer intends to use? If not I'd just simply the ToC entry here to 'Create compute'

Reply:
Updated compute section.